lua/class_definitions.lua

```diff
@@ -84,11 +84,6 @@ classes = {
             nv_cached = { type = "bool", writable = true },
         },
         functions = {
-            { name = "add_effect", rval = nil, args = { "string", "int" } },
```

Comment:
> My mistake, there was a version on the forum that uses add_effect. http://smf.cataclysmdda.com/index.php?topic=11353.0

My mistake as well. The Lua wrapper *does* understand string ids, so the functions can be used in Lua. I added them back in with the proper string id type as parameter. The mod needs to be changed to call the functions with the wrapper:
```Lua
-- Instead of
player:has_effect("hot")
-- One has to do this:
player:has_effect(efftype_id("hot"))
```